packages/core/src/icon/icon.element.spec.ts

```diff
@@ -18,7 +18,7 @@ describe('icon element', () => {
   let component: CdsIcon;
 
   beforeAll(() => {
-    ClarityIcons.add({ testing: testIcon });
+    ClarityIcons.addIcons(['testing', testIcon]);
```

Comment:
In this file, the `.clr-i-size-*` unit tests are still here, despite being removed from CSS.

Please correct.